src/single-spa-vue.js

```diff
@@ -88,14 +88,17 @@ function mount(opts, mountedInstances, props) {
           }
         } else {
           domEl = appOptions.el;
+          if (!domEl.parentNode) {
+            throw Error(
+              `If appOptions.el is provided to single-spa-vue, the dom element must exist in the dom. Was provided as ${appOptions.el.outerHTML}`
```

Comment:
It might be valid to mount to a disconnected DOM node, so I'm not sure this if statement and error are necessary

Reply:
So the reason why we added this `if` is that we wanted to keep the behaviour aligned with the case where `appOptions.el` is a `string`.

If you think it is not necessary, I can remove it.
Let me know.